spark-env.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-export SPARK_EXECUTOR_URI=http://pele.farmbio.uu.se/spark-mantl/spark-mantl-1.6.0.tgz
+export SPARK_EXECUTOR_URI=http://pele.farmbio.uu.se/spark-mantl/spark-mantl-2.0.1.tgz
```

Comment:
Hi @yanivr78 thanks for your PR. I am afraid there is no spark-mantl-2.0.1.tgz in our server. That's a University machine, and I am afraid I can't put a new version of Spark, since we don't maintain the MANTL deployment anymore. The reason why we need a custom Spark distribution, is that MANTL uses Mesos authentication. You need to download http://pele.farmbio.uu.se/spark-mantl/spark-mantl-1.6.0.tgz and look for the configuration that I made in the conf directory, and then you need preapare a new TGZ with Spark 2.0.1. Also you should provide hosting for this file.